addons/assistance.py

```diff
@@ -670,10 +670,10 @@ def check_console(self, message, channel, consoles):
     @tutorial.command(hidden=True, )
     async def randomize(self):
         """Links to layeredfs randomizing tutorial"""
-        embed = discord.Embed(title="randomizing with layeredfs", color=discord.Color.red())
+        embed = discord.Embed(title="Randomizing with LayeredFS", color=discord.Color.red())
         embed.set_thumbnail(url="https://i.imgur.com/rr7Xf3E.jpg")
         embed.url = "https://zetadesigns.github.io/randomizing-layeredfs.html"
-        embed.description = "Basic tutorial for randomizing with layeredfs"
+        embed.description = "Basic tutorial for Randomizing with LayeredFS"
```

Comment:
shouldnt `randomizing`  be lowercase here?

Reply:
Hmm yeah. ~~Might have confused that on accident~~